libraries/ui/src/HamburgerButton.tsx

```diff
@@ -0,0 +1,40 @@
+import clsx from 'clsx';
+import React from 'react';
+
+const HamburgerIcon: React.FC<{ className?: string }> = ({ className }) => (
+  <svg xmlns="http://www.w3.org/2000/svg" width="48" height="48" viewBox="0 0 48 48" fill="none" className={className}>
+    <path d="M16 18H32" stroke="currentColor" strokeWidth="2" strokeLinecap="round" />
+    <path d="M16 24H32" stroke="currentColor" strokeWidth="2" strokeLinecap="round" />
+    <path d="M16 30H32" stroke="currentColor" strokeWidth="2" strokeLinecap="round" />
+  </svg>
+);
+
+const CloseIcon: React.FC<{ className?: string }> = ({ className }) => (
+  <svg xmlns="http://www.w3.org/2000/svg" width="48" height="48" viewBox="0 0 48 48" fill="none" className={className}>
+    <path d="M18 29.3135L29.3137 17.9998" stroke="currentColor" strokeWidth="2" strokeLinecap="round" />
+    <path d="M18 18.3135L29.3137 29.6272" stroke="currentColor" strokeWidth="2" strokeLinecap="round" />
+  </svg>
+);
```

Comment:
I've just defined these and other svgs inline, pending a more permanent solution to how we handle them. I could at least move them to an `/icons` folder if that would be a step in the right direction